config/v1/types_network.go

```diff
@@ -39,6 +39,12 @@ type NetworkSpec struct {
 	// Currently supported values are:
 	// - OpenShiftSDN
 	NetworkType string `json:"networkType"`
+
+	// externalIP is the policy applied to the ExternalIP field in a Service.
+	// If nil, any value is allowed for an ExternalIP. If the empty/zero
+	// policy is supplied, then ExternalIP is not allowed to be set.
+	// +optional
+	ExternalIPs *ExternalIPPolicy `json:"externalIPs,omitempty"`
```

Comment:
Why is this being added to `config.openshift.io`'s NetworkSpec rather than `operator.openshift.io`'s? And even if it's going in the former, shouldn't it get copied to the latter too for consistency? (Otherwise it's the only piece of data that is in the `config` object but not the `operator` object.) And if it does go in the `operator` config, it seems like it would go in `spec.kubeProxyConfig`, in which case maybe there should be something parallel in the `config` object, in case we want to expose more proxy config later...

Reply:
(Answered Dan's question elsewhere, but bringing the answer here)

Because this isn't consumed by the network operator. Rather, it's observed by the cluster-kube-apiserver-operator. So, it needs to be cluster-level configuration.